seed/urls/main.py

```diff
@@ -151,6 +153,43 @@
         name='delete_organization_buildings'
     ),
 
+    # timeseries data
```

Comment:
@pipermerriam specifically on this endpoint or the timeseries feature
overall?

Rich Brown
Building Technology and Urban Systems Department
Lawrence Berkeley National Laboratory
Sent from my mobile phone

On Feb 15, 2016, at 7:34 AM, Piper Merriam <notifications@github.com> wrote:

In seed/urls/main.py
<https://github.com/SEED-platform/seed/pull/714#discussion_r52911948>:

> @@ -151,6 +153,43 @@
>          name='delete_organization_buildings'
>      ),
>
> +    # timeseries data

I have a hard time having helpful feedback on this because I don't have a
lot of context for what this is doing or the functionality it is supporting.

—
Reply to this email directly or view it on GitHub
<https://github.com/SEED-platform/seed/pull/714/files#r52911948>.